### ISSUE TITLE
Rename "VID" throughout the app to "VC"

### ISSUE THREAD

**Issue by kneckinator** [idpass/inji#18]
Given that it will be possible to retrieve a Verifiable Credential (VC) either from a UIN or a VID, we should prepare for this by using "VC" to represent the object that is downloaded and shared.

We currently aligned on the term "VID" to represent the verifiable credential which can be confusing to developers.

The current term works fine when the only base for a verifiable credential is the UIN.

**Comment by kneckinator** [comment]
Now that the display string for "VID" is configurable on the "settings" screen of the app, I believe this will only be about changing the default of that setting @pmigueld ?

On top of this, creating a compound string for things like "enter UIN" to change them to "enter UIN or {vid}"

**Comment by pmigueld** [comment]
@kneckinator Yes, we can just update the default setting. However we still refer to them as `vid` in the code so we should also update them eventually to be more clear right? Because we currently have a case of referencing the UIN of a VID (`vid.uin`) which no longer seems to make sense?

**Comment by kneckinator** [comment]
Yes we can raise a ticket to clean up the code later.

**Comment by celinenilla** [comment]
Tested on 0.2.1-beta2-newlogic_20220303_1026.apk. Working as expected. There are still two places where "VID" is mentioned, see #17
